pkg/operator/ceph/object/controller.go

```diff
@@ -351,7 +351,9 @@ func (r *ReconcileCephObjectStore) reconcile(request reconcile.Request) (reconci
 
 	// update ObservedGeneration in status at the end of reconcile
 	// Set Progressing status, we are done reconciling, the health check go routine will update the status
-	updateStatus(r.opManagerContext, observedGeneration, r.client, request.NamespacedName, cephv1.ConditionProgressing, buildStatusInfo(cephObjectStore))
+	updateStatus(r.opManagerContext, observedGeneration, r.client, request.NamespacedName, cephv1.ConditionReady, buildStatusInfo(cephObjectStore))
+	// TODO: discuss above: is "Ready" the right condition here? Should we wait in a loop to check
+	// for RGW response beforehand?
```

Comment:
Todo: discuss this

Reply:
If we're removing the health check, sounds good just to set this to Ready